README.md

```diff
@@ -1,2 +1,202 @@
-# saritasa-github-actions
-GitHub actions
+## CI-wordpress reusable workflow
+
+### Deploying
+
+In order to deploy, you need to perform some steps:
+
+- Create a new playbook for deploying your app, for example, `deploy.yml`
+- Place the created playbook in your repository for example, `ci\ansible\deploy.yaml`
+- Set up variables for workflow (see Inputs Variables)
+- Create a workflow, configure workflow start triggers (see examles)
+
+If everything has been set up properly, after completing the workflow approximately the following structure will be created on the server.
+Check how the on your server folder structure would look like after one, two and three deployments.
+
+```
+-- /var/www/my-app.com
+|-- current -> /var/www/my-app.com/releases/20100509145325
+|-- releases
+|   |-- 20100509145325
+|-- shared
+```
+
+```
+-- /var/www/my-app.com
+|-- current -> /var/www/my-app.com/releases/20100509150741
+|-- releases
+|   |-- 20100509150741
+|   |-- 20100509145325
+|-- shared
+```
+
+```
+-- /var/www/my-app.com
+|-- current -> /var/www/my-app.com/releases/20100512131539
+|-- releases
+|   |-- 20100512131539
+|   |-- 20100509150741
+|   |-- 20100509145325
+|-- shared
+```
+
+### Rolling back
+
+In order to rollback, you need to set up the deployment and run the rollback workflow. Workflow, will switch the current folder to the previous release.
+
+Before rollback
+```
+-- /var/www/my-app.com
+|-- current -> /var/www/my-app.com/releases/20100512131539
+|-- releases
+|   |-- 20100512131539
+|   |-- 20100509150741
+|   |-- 20100509145325
+|-- shared
+```
+
+After rollback
+```
+-- /var/www/my-app.com
+|-- current -> /var/www/my-app.com/releases/20100509150741
```

Comment:
why release is some random ID? Is it timestamp? If so can we organize the solution to have a human-readable timestamp? i.e. something like

```
releases/2023-09-23-1695930445
```
i.e. YYYY-MM-DD-UNIXTIMESTAMP

Reply:
done
![image](https://github.com/saritasa-nest/saritasa-github-actions/assets/114897334/4bd629ec-9a4d-4790-9263-7e7778433adf)
Rollback also works
But apart from convenient reading, this does not give anything, anstrano uses `ls-lt` to build a timeline of releases when rolling back